canvas_modules/common-canvas/locales/common-properties/locales/en.json

```diff
@@ -75,6 +75,8 @@
   "expression.maximize.label": "Maximize",
   "toggle.on.label": "On",
   "toggle.off.label": "Off",
+  "password.showPasswordLabel.tooltip":"Show",
+  "password.hidePasswordLabel.tooltip":"Custom Hide Value",
```

Comment:
Locales under common-properties are the default if they are not overridden by the application. This should be `Show` and `Hide`